moveit_core/constraint_samplers/include/moveit/constraint_samplers/constraint_sampler.h

```diff
@@ -115,7 +115,9 @@ class ConstraintSampler
    *
    * @return The planning scene as a const ptr
    */
-  const planning_scene::PlanningSceneConstPtr& getPlanningScene() const
+  [[deprecated("getPlanningScene() is deprecated, use LockedPlanningSceneRO or LockedPlanningSceneRW "
+               "instead.")]] const planning_scene::PlanningSceneConstPtr&
+  getPlanningScene() const
```

Comment:
The deprecation doesn't make sense in this context, since there is no PSM provided by this class that could be locked.